monkey/monkey_island/cc/server_utils/setup_flask.py

```diff
@@ -0,0 +1,24 @@
+import json
+import secrets
+from pathlib import Path
+from typing import Any, Dict
+
+from common.utils.file_utils import open_new_securely_permissioned_file
+
+SECRET_FILE_NAME = ".flask_security_configuration.json"
+
+
+def generate_flask_security_configuration(data_dir: Path) -> Dict[str, Any]:
+    SECRET_FILE_PATH = str(data_dir / SECRET_FILE_NAME)
+    try:
+        with open(SECRET_FILE_PATH, "r") as secret_file:
+            return json.load(secret_file)
+    except FileNotFoundError:
+        with open_new_securely_permissioned_file(SECRET_FILE_PATH, "w") as secret_file:
+            secret_key = secrets.token_urlsafe(32)
```

Comment:
`SECURITY_KEY` doesn't need to be stored. We can generate it every time the Island is run. This is only going to be used for sessions as far as I can tell.

Reply:
https://stackoverflow.com/questions/34902378/where-do-i-get-secret-key-for-flask#comment128528133_57936977

I think we want to preserve the session, @VakarisZ ?

Reply:
@shreyamalviya you're right, it's only used for sessions. The benefits of having the same `security_key` is that we don't need to worry about invalidating the sessions on island restart. What is something kills the island process and now the db is populated with invalid sessions. What will flask-login do? throw errors? We then need a step to clean-up all sessions on startup. Other upside is that island can be restarted without the need to login again.

Reply:
> What if something kills the island process 

Good point. Agents will also not be able to pull the latest data from the Island then (propagation credentials, other instructions once the Island is made a proper C&C server).